cdap-ui/app/directives/instance-control/instance-control.js

```diff
@@ -16,22 +15,12 @@ angular.module(PKG.name + '.commons')
         scope.handleSet = function () {
           scope.processing = true;
 
-          if (scope.model.requested === undefined || scope.model.requested < 0) {
-            myAlert({
-              title: 'Invalid instances: ',
-              content: 'you must request a valid number of instances.',
-              type: 'danger'
-            });
-            scope.processing = false;
-            return;
-          }
-
           scope.myDataSrc.request({
```

Comment:
If we are having a separate controller for GETing the instances, shouldn't this POST be moved to the same controller? Link functions are generally used to hook any event listeners on DOM elements (generally, could be used for multiple different things but in general used for accessing the DOM). Thoughts?